### PR TITLE
feat: v0.0.2  huge front update!

### PR DESCRIPTION
**Pull request by enescomruk72**
That was huge fail! sorry about that! this is last one, I swear it.

### CODE REVIEW

File: frontend/src/assets/css/style.css [hunk 1]
```diff
@@ -10,57 +10,55 @@ body {
...
+        --muted: 60 4.8% 95.9%;
+        --muted-foreground: 25 5.3% 44.7%;
+        --accent: 60 4.8% 95.9%;
+        --accent-foreground: 24 9.8% 10%;
+        --destructive: 0 84.2% 60.2%;
+        --destructive-foreground: 60 9.1% 97.8%;
+        --border: 20 5.9% 90%;
+        --input: 20 5.9% 90%;
+        --ring: 20 14.3% 4.1%;
+        --radius: 0.5rem;
```
Comment (CoderMungan) [inline frontend/src/assets/css/style.css:35]: best!

File: frontend/src/assets/css/style.css [hunk 2]
```diff
@@ -10,57 +10,55 @@ body {
...
+        --muted: 12 6.5% 15.1%;
+        --muted-foreground: 24 5.4% 63.9%;
+        --accent: 12 6.5% 15.1%;
+        --accent-foreground: 60 9.1% 97.8%;
+        --destructive: 0 62.8% 30.6%;
+        --destructive-foreground: 60 9.1% 97.8%;
+        --border: 12 6.5% 15.1%;
+        --input: 12 6.5% 15.1%;
+        --ring: 24 5.7% 82.9%;
+    }
```
Comment (CoderMungan) [inline frontend/src/assets/css/style.css:58]: best!

File: frontend/src/components/icons/logo.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { cn } from "@/lib/utils";
+import logo from "./logo.png";
+export const Logo = ({ className }) => {
+  return <img src={logo} alt="Rythm Technologies Logo" className={className} />;
+};
```
Comment (CoderMungan) [inline frontend/src/components/icons/logo.jsx:5]: best!

File: frontend/src/components/projects.jsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            </a>
+            <p className="text-center font-bold text-foreground">
+              {project.name}
+            </p>
+          </div>
+        ))}
+      </div>
+    </article>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/components/projects.jsx:63]: best!

File: frontend/src/components/spinner/index.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        <path
+          className="opacity-75"
+          fill="currentColor"
+          d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
+        ></path>
+      </svg>
+      <span className="sr-only">Loading</span>
+    </>
+  );
+};
```
Comment (CoderMungan) [inline frontend/src/components/spinner/index.jsx:51]: best!

File: frontend/src/components/ui/card.jsx [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+CardFooter.displayName = "CardFooter";
+
+export {
+  Card,
+  CardHeader,
+  CardFooter,
+  CardTitle,
+  CardDescription,
+  CardContent,
+};
```
Comment (CoderMungan) [inline frontend/src/components/ui/card.jsx:67]: best!

File: frontend/src/components/ui/form.jsx [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+export {
+  useFormField,
+  Form,
+  FormItem,
+  FormLabel,
+  FormControl,
+  FormDescription,
+  FormMessage,
+  FormField,
+}
```
Comment (CoderMungan) [inline frontend/src/components/ui/form.jsx:133]: great work!

File: frontend/src/components/ui/heading.jsx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        ref={ref}
+        {...props}
+        children={children}
+      />
+    );
+  }
+);
+Heading.displayName = "Heading";
+
+export { Heading, headingVariants };
```
Comment (CoderMungan) [inline frontend/src/components/ui/heading.jsx:52]: great work!

File: frontend/src/components/ui/input.jsx [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+        "flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-sm shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50",
+        className
+      )}
+      ref={ref}
+      {...props} />)
+  );
+})
+Input.displayName = "Input"
+
+export { Input }
```
Comment (CoderMungan) [inline frontend/src/components/ui/input.jsx:19]: great work!

File: frontend/src/components/ui/label.jsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+const labelVariants = cva(
+  "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+)
+
+const Label = React.forwardRef(({ className, ...props }, ref) => (
+  <LabelPrimitive.Root ref={ref} className={cn(labelVariants(), className)} {...props} />
+))
+Label.displayName = LabelPrimitive.Root.displayName
+
+export { Label }
```
Comment (CoderMungan) [inline frontend/src/components/ui/label.jsx:16]: great work!

File: frontend/src/components/ui/mode-toggle.jsx [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+              )}
+            >
+              {<theme.icon width="20" />} {theme.value}
+            </DropdownMenuItem>
+          );
+        })}
+      </DropdownMenuContent>
+    </DropdownMenu>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/components/ui/mode-toggle.jsx:68]: you best Jhonyyy!

File: frontend/src/components/ui/form.jsx (hunk 7 above)
Comment (enescomruk72) [reply frontend/src/components/ui/form.jsx:133]: thx!

File: frontend/src/components/ui/navigation-menu.jsx [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+  navigationMenuTriggerStyle,
+  NavigationMenu,
+  NavigationMenuList,
+  NavigationMenuItem,
+  NavigationMenuContent,
+  NavigationMenuTrigger,
+  NavigationMenuLink,
+  NavigationMenuIndicator,
+  NavigationMenuViewport,
+}
```
Comment (CoderMungan) [inline frontend/src/components/ui/navigation-menu.jsx:104]: you best Jhonyyy!

File: frontend/src/components/ui/skeleton.jsx [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+  ...props
+}) {
+  return (
+    (<div
+      className={cn("animate-pulse rounded-md bg-primary/10", className)}
+      {...props} />)
+  );
+}
+
+export { Skeleton }
```
Comment (CoderMungan) [inline frontend/src/components/ui/skeleton.jsx:14]: king of jhoonnnyyy!

File: frontend/src/components/ui/mode-toggle.jsx (hunk 11 above)
Comment (enescomruk72) [reply frontend/src/components/ui/mode-toggle.jsx:68]: Yea janıt! thx

File: frontend/src/components/ui/tabs.jsx [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+    className={cn(
+      "mt-2 ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2",
+      className
+    )}
+    {...props}
+  />
+));
+TabsContent.displayName = TabsPrimitive.Content.displayName;
+
+export { Tabs, TabsList, TabsTrigger, TabsContent };
```
Comment (CoderMungan) [inline frontend/src/components/ui/tabs.jsx:44]: king of jhoonnnyyy!

File: frontend/src/components/ui/textarea.jsx [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+        "flex min-h-[60px] w-full rounded-md border border-input bg-transparent px-3 py-2 text-sm shadow-sm placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50",
+        className
+      )}
+      ref={ref}
+      {...props} />)
+  );
+})
+Textarea.displayName = "Textarea"
+
+export { Textarea }
```
Comment (CoderMungan) [inline frontend/src/components/ui/textarea.jsx:18]: king of jhoonnnyyy!

File: frontend/src/layouts/web/components/header/components/HeaderLogo.jsx [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+        className="hidden font-bold sm:inline-block whitespace-nowrap"
+      >
+        Rythm{" "}
+        <Heading level="h5" className=" inline-flex text-muted-foreground">
+          Technologies
+        </Heading>
+      </Heading>
+    </Link>
+  );
+};
```
Comment (CoderMungan) [inline frontend/src/layouts/web/components/header/components/HeaderLogo.jsx:20]: king of jhoonnnyyy!

File: frontend/src/layouts/web/components/header/components/ListItem.jsx [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+            <p className="line-clamp-2 text-sm leading-snug text-muted-foreground">
+              {children}
+            </p>
+          </Link>
+        </NavigationMenuLink>
+      </li>
+    );
+  }
+);
+ListItem.displayName = "ListItem";
```
Comment (CoderMungan) [inline frontend/src/layouts/web/components/header/components/ListItem.jsx:32]: Strom king Jhony Enesss!

Comment (enescomruk72) [reply frontend/src/layouts/web/components/header/components/ListItem.jsx:32]: Volibeaaar uhaauhaa

File: frontend/src/layouts/web/components/header/components/Navbar.jsx [hunk 18]
```diff
@@ -0,0 +1,195 @@
...
+
+  return (
+    <NavigationMenu className="container hidden md:flex max-w-3xl mx-auto bg-transparent text-sm">
+      <NavigationMenuList>
+        <DropDownLinks dropDownLinks={dropDownLinks} />
+        <ClassicNavLinks classicLinks={classicLinks} />
+      </NavigationMenuList>
+    </NavigationMenu>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/layouts/web/components/header/components/Navbar.jsx:195]: React tell me about u "Gimme mercy Enes"

File: frontend/src/layouts/web/components/header/index.jsx [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+          <HeaderLogo />
+          <Navbar />
+        </div>
+        <div className="flex flex-1 items-center space-x-2 justify-end">
+          <ModeToggle />
+        </div>
+      </div>
+    </header>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/layouts/web/components/header/index.jsx:19]: React tell me about u "Gimme mercy Enes"

File: frontend/src/layouts/web/components/header/components/Navbar.jsx (hunk 18 above)
Comment (enescomruk72) [reply frontend/src/layouts/web/components/header/components/Navbar.jsx:195]: I forgive u

File: frontend/src/layouts/web/components/subscribe/index.jsx [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+                </FormDescription>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
+        </form>
+      </Form>
+    </section>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/layouts/web/components/subscribe/index.jsx:81]: React tell me about u "Gimme mercy Enes"

File: frontend/src/layouts/web/index.jsx [hunk 21]
```diff
@@ -16,11 +21,40 @@ export default function WebLayout() {
...
+        />
+      </div>
+      {/* <div className="z-10 text-center px-8 drop-shadow-lg shadow-black">
+        <div className="uppercase text-sm mb-4">Welcome to</div>
+        <div className="text-4xl font-mplus font-medium">
+          A curated list of the tech I{" "}
+          <span className="text-orange-500">use</span>
+        </div>
+      </div> */}
+    </section>
```
Comment (CoderMungan) [inline frontend/src/layouts/web/index.jsx:58]: Crazy Enes Worked on!

File: frontend/src/pages/about/index.jsx [hunk 22]
```diff
@@ -1,3 +1,82 @@
...
+              </CardDescription>
+            </CardHeader>
+            <CardContent>
+              <div dangerouslySetInnerHTML={{ __html: about.content }} />
+            </CardContent>
+          </Card>
+        </TabsContent>
+      ))}
+    </Tabs>
+  );
```
Comment (CoderMungan) [inline frontend/src/pages/about/index.jsx:81]: Crazy Enes Worked on!

File: frontend/src/pages/contact/index.jsx [hunk 23]
```diff
@@ -1,3 +1,136 @@
...
+              />
+              <FormItem className="text-end">
+                <Button type="submit">Submit</Button>
+              </FormItem>
+            </form>
+          </Form>
+        </div>
+      </main>
+    </div>
+  );
```
Comment (CoderMungan) [inline frontend/src/pages/contact/index.jsx:135]: Best software engineer from KahramanMaraş!

File: frontend/src/pages/not-found/index.jsx [hunk 24]
```diff
@@ -0,0 +1,24 @@
...
+        </CardDescription>
+        <Link to="/">
+          <Button variant="contained" color="primary">
+            Ana Sayfaya Dön
+          </Button>
+        </Link>
+      </Card>
+    </div>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/pages/not-found/index.jsx:24]: Best software engineer from KahramanMaraş!

File: frontend/src/pages/projects/index.jsx [hunk 25]
```diff
@@ -0,0 +1,18 @@
...
+        level="h1"
+        className="scroll-m-20 text-3xl font-bold tracking-tight mb-4 px-4"
+      >
+        Projects
+      </Heading>
+      <ProjecstSection />
+      <Outlet />
+    </>
+  );
+}
```
Comment (CoderMungan) [inline frontend/src/pages/projects/index.jsx:18]: good job!

**Review by CoderMungan — APPROVED** [review]
Great Fucking Job! Let's Go!!!

**Pull request merged**